src/openslide-decode-tiff.c

```diff
@@ -483,6 +483,51 @@ bool _openslide_tiff_add_associated_image(openslide_t *osr,
   return ret;
 }
 
+bool _openslide_tiff_get_icc_profile_size(struct _openslide_tiff_level *tiffl,
+                                          TIFF *tiff,
+                                          uint32_t *icc_profile_size,
```

Comment:
If we make this an `int64_t *`, we can avoid a temporary variable in every caller.

Reply:
Done.